master-test-cases.yml

```diff
@@ -141,14 +141,12 @@ body:
         - '{"value":null}'
 
   # named types
+  # TODO(dfox): add more positive/negative cases when https://github.com/palantir/conjure/issues/193 is resolved 
```

Comment:
I've written up the options in https://github.com/palantir/conjure/issues/193 for posterity

Reply:
thanks!